pkg/app/web/src/components/settings-page/index.tsx

```diff
@@ -76,6 +82,13 @@ export const SettingsIndexPage: FC = memo(function SettingsIndexPage() {
                 activeClassName={classes.activeNav}
               >
                 <ListItemText primary={text} />
+                {link === PAGE_PATH_SETTINGS_ENV && (
```

Comment:
not sure why do we need this check? 🤔

Reply:
Because this is only for the Environment tab, but am I correct?

Reply:
If this condition check exists, when we click on other tabs on the left sidebar, the warning will disappear. I think it should keep be shown even if users click on other tabs on the left sidebar, wdyt?

<img width="281" alt="Screen Shot 2022-01-12 at 18 50 54" src="https://user-images.githubusercontent.com/32532742/149116997-5fe16154-a15c-4f01-8938-cd524d54a216.png">

Reply:
The variable name may be a bit confusing, but it just tells it to show the Icon only if it is a ListItem for Environment.

Reply:
Ah okay, I get the point 👍 Thanks you.